### PR TITLE
add bridge marker pod description

### PR DESCRIPTION
**Pull request by RamLavi**
This PR adds bridge marker pod description

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: manifests/bridge-marker.yml.in [hunk 1]
```diff
@@ -16,6 +16,8 @@ spec:
         tier: node
         app: bridge-marker
         name: bridge-marker
+      annotations:
+        description: Bridge marker pod marks network bridges available on nodes as node resources
```
Comment (phoracek) [inline manifests/bridge-marker.yml.in:20]: Bridge marker exposes network bridges available on nodes as node resources

So we don't mark mark

**Review by phoracek — CHANGES_REQUESTED** [review]

File: manifests/bridge-marker.yml.in (hunk 1 above)
Comment (RamLavi) [reply manifests/bridge-marker.yml.in:20]: ack,
Done

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

**Pull request merged**